src/studio/src/designer/frontend/shared/features/dataModelling/components/CreateNewWrapper.tsx

```diff
@@ -93,6 +93,7 @@ export function CreateNewWrapper(props: ICreateNewWrapper) {
       </TopToolbarButton>
       {createButtonAnchor && (
         <AltinnPopoverSimple
+          open={!!createButtonAnchor}
```

Comment:
Is this not implied by the attribute below?

Reply:
Tests failed because of not setting this explisitt. Not sure how to complete explain this. But in short the `useRef`-hook need to be mocked out for some MUI-stuff to work, and that again is breaking the tests. I guess there is a reason for the underlaying Popover-component from MUI to have `open` as an explisitt property.